src/main/resources/beam-template.conf

```diff
@@ -144,7 +144,7 @@ beam.agentsim.agents.rideHail.allocationManager.repositionLowWaitingTimes.waitin
 beam.agentsim.agents.rideHail.allocationManager.repositionLowWaitingTimes.demandWeight = "double | 4.0"
 beam.agentsim.agents.rideHail.allocationManager.repositionLowWaitingTimes.produceDebugImages = true
 beam.agentsim.agents.rideHail.allocationManager.alonsoMora.pickupTimeWindowInSec = "int | 360"
-beam.agentsim.agents.rideHail.allocationManager.alonsoMora.dropoffTimeWindowInSec = "int | 600"
+beam.agentsim.agents.rideHail.allocationManager.alonsoMora.dropoffTimeWindowInSec= "double | 0.2"
```

Comment:
this seems too low 0.2 seconds...

Reply:
It has been changed into a multiplier in a linear model. But it should have a new name.

Reply:
ok, got it